mono/metadata/unity-liveness.c

```diff
@@ -231,177 +248,162 @@ static gboolean mono_field_can_contain_references(MonoClassField* field)
 }
 
-static gboolean should_traverse_objects (size_t index, gint32 recursion_depth)
+static gboolean should_traverse_objects(size_t index, gint32 recursion_depth)
 {
 	// Add kArrayElementsPerChunk objects at a time and then traverse
-	return ((index + 1) & (kArrayElementsPerChunk - 1)) == 0 && 
+	return ((index + 1) & (kArrayElementsPerChunk - 1)) == 0 &&
 		recursion_depth < kMaxTraverseRecursionDepth;
 }
 
-static void mono_traverse_array (MonoArray* array, LivenessState* state)
+static void mono_traverse_array(MonoArray *array, LivenessState *state)
 {
 	size_t i = 0;
 	gboolean has_references;
-	MonoObject* object = (MonoObject*)array;
-	MonoClass* element_class;
-	size_t elementClassSize;
+	MonoObject *object = (MonoObject *)array;
+	MonoClass *element_class;
+	int32_t elementClassSize;
 	size_t array_length;
-	
-	g_assert (object);
-	
-	
-	
+
+	g_assert(object);
+
 	element_class = GET_VTABLE(object)->klass->element_class;
 	has_references = !mono_class_is_valuetype(element_class);
 	g_assert(element_class->size_inited != 0);
-	
-	for (i = 0; i < mono_class_get_field_count (element_class); i++)
-	{
+
+	for (i = 0; i < mono_class_get_field_count(element_class); i++) {
 		has_references |= mono_field_can_contain_references(&element_class->fields[i]);
 	}
-	
+
 	if (!has_references)
 		return;
-	
-	array_length = mono_array_length (array);
-	if (element_class->valuetype)
-	{
+
+	array_length = mono_array_length(array);
+	if (element_class->valuetype) {
 		size_t items_processed = 0;
-		elementClassSize = mono_class_array_element_size (element_class);
-		for (i = 0; i < array_length; i++)
-		{
-			MonoObject* object = (MonoObject*)mono_array_addr_with_size (array, elementClassSize, i);
-			if (mono_traverse_object_internal (object, 1, element_class, state))
+		elementClassSize = mono_class_array_element_size(element_class);
+		for (i = 0; i < array_length; i++) {
+			MonoObject *object = (MonoObject *)mono_array_addr_with_size(array, elementClassSize, i);
+			if (mono_traverse_object_internal(object, 1, element_class, state))
 				items_processed++;
-			
-			if(should_traverse_objects (items_processed, state->traverse_depth))
+
+			if (should_traverse_objects(items_processed, state->traverse_depth))
 				mono_traverse_objects(state);
 		}
 	}
-	else
-	{
+	else {
 		size_t items_processed = 0;
-		for (i = 0; i < array_length; i++)
-		{
-			MonoObject* val =  mono_array_get(array, MonoObject*, i);
-			if (mono_add_process_object (val, state))
+		for (i = 0; i < array_length; i++) {
+			MonoObject *val = mono_array_get(array, MonoObject *, i);
+			if (mono_add_process_object(val, state))
 				items_processed++;
-			
-			if (should_traverse_objects (items_processed, state->traverse_depth))
+
+			if (should_traverse_objects(items_processed, state->traverse_depth))
 				mono_traverse_objects(state);
 		}
 	}
 }
 
-
-void mono_filter_objects(LivenessState* state)
+void mono_filter_objects(LivenessState *state)
 {
 	gpointer filtered_objects[64];
```

Comment:
Does 64 still make sense here?

Reply:
didn't change this, but 64 is the batch size sent to unity for registering